packages/next-templates/src/components/PreFooterNav/PreFooterNav.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link, BreadcrumbNavSeparator } from '@utrecht/component-library-react';
+import { IconChevronRight, IconChevronUp } from '@tabler/icons-react';
+import './pre-footer-nav.css';
+
+export const PreFooterNav = () => (
+  <div className="voorbeeld-pre-footer">
+    <Link className="voorbeeld-pre-footer-link" href="/404">
```

Comment:
In BEM is dit `voorbeeld-pre-footer__link`.

Het systeem is: `[prefix]-[block]__[element]--[modifier]`

Reply:
Aangepast! 
Hier nog even extra mee oefenen.😌